script.js

```diff
@@ -0,0 +1,19 @@
+var plength = document.getElementById("myRange").value;
+//var defaultLength = plength.defaultvalue;
+//var pickLength = plength.value;
+
+var alphnum = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789!#$%&'()*+,-./:;<=>?@[]^_`{|}~";
+var newpassword = "";
+
+
+function genpassword(){
+
```

Comment:
I would put var newpassword here so that on every generation the password is being empty and then created anew instead of adding it to the previous iteration